docs/user-guide/getting-started.md

```diff
@@ -1,185 +1,202 @@
 ---
-    description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
+   description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
 ---
 # Quick Start Installation Guide
 
 In this guide, we'll walk you through installing and configuring the Ambassador Edge Stack in your Kubernetes cluster. Within a few minutes, your cluster will be routing HTTPS requests from the Internet to a backend service. You'll also have a sense of how the Ambassador Edge Stack is managed.
 
+Different options for installation include:
+
+* **[Quick Install (recommended!)](#quick-install)**
+* [Install via Minikube](#minikube-users)
+* [Install in CI](#install-in-ci)
+
+Or, you can [install manually](/user-guide/manual-install) if you want to
+customize your configuration.
+
 ## Before You Begin
 
 The Ambassador Edge Stack is designed to run in Kubernetes for production. The most essential requirements are:
 
 * Kubernetes 1.11 or later
 * The `kubectl` command-line tool
+* [Edge Control](/reference/edgectl-download)
+
+## Quick Install (Recommended!)
+
+The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally. Or, if you're a Minikube user, [check out these directions](#minikube-users).
 
-## Install the Ambassador Edge Stack
+When you use Edge Control on your publicly
+accessible cluster, it will:
 
-The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally.
+1. Install the Ambassador Edge Stack
+2. Generate a domain name for you to access the Edge Policy Console and complete
+   advanced configuration
+3. Obtain a TLS certificate for that domain name
+4. Configure automatic TLS and HTTPS using that certificate
 
-1. In your terminal, run the following command:
+Install Edge Control for your operating system [here](/reference/edgectl-download). When complete, return to these instructions.
```

Comment:
I dislike using "[here]" in any prose: it's not a natural English phrasing. Better would be "Install Edge Control for your operating system by following [these instructions]." That's a better English phrasing.

But more so, I'm objecting to this sending the user to another page and then asking them to return to this page: that's just adding friction to the process and the whole point of this easy installation is to eliminate friction. We lose readers with every click and with every step, so the goal is to minimize clicks and minimize steps.

Reply:
I agree, we should minimize clicks and can easily combine the two pages. How do you envision the layout? I very strongly believe that since it's such a hot page SEO and traffic wise, we do need to consult @richarddli or @kelseyevans about what kind of content should or shouldn't be there.

Reply:
I agree, the Quickstart page should contain all the information you need to get started (in the way we recommend) without having to click to another page.

Reply:
SEO-wise, this doesn't raise any flags. With this page we're most concerned about satisfying user intent.